intake_thredds/source.py

```diff
@@ -69,6 +69,8 @@ def __init__(
         self.urlpath = url
         if 'simplecache::' in url:
             self.metadata.update({'fsspec_pre_url': 'simplecache::'})
+        assert isinstance(path, list), print('path must be list of str')
+        assert isinstance(path[0], str), print('path must be list of str')
```

Comment:
The print function isn't necessary here

```suggestion
        assert isinstance(path, list), 'path must be list of str'
        assert isinstance(path[0], str), 'path must be list of str'
```